scarb/src/ops/fmt.rs

```diff
@@ -21,15 +21,18 @@ pub struct FmtOptions {
 #[tracing::instrument(skip_all, level = "debug")]
 pub fn format(opts: FmtOptions, ws: &Workspace<'_>) -> Result<bool> {
     ws.config().ui().force_colors_enabled(opts.color);
-    let config = FormatterConfig::default();
-    let fmt = CairoFormatter::new(config);
+
     let packages = ws
         .members()
         .filter(|pkg| opts.packages.contains(&pkg.id))
         .collect::<Vec<_>>();
     let Some((first_package, packages)) = packages.split_first() else {
         return Ok(true);
     };
+
+    let config = first_package.manifest.fmt_config.clone().into();
+    let fmt = CairoFormatter::new(config);
```

Comment:
There's a bug here: what if particular packages declare their own formatter configs? I think we should sequentially walk over all packages here. It's a tradeoff, but it shouldn't be so severe I think.